types/packages/caver-klay/src/index.d.ts

```diff
@@ -86,7 +86,7 @@ export class DeprecatedKlayRPC {
     getHeaderByHash: RpcCallToMethod['klay_getHeaderByHash']
     getBlockReceipts: RpcCallToMethod['klay_getBlockReceipts']
     getRewards: RpcCallToMethod['klay_getRewards']
-    getChainConfig: RpcCallToMethod['klay_chainConfig']
+    getChainConfig: RpcCallToMethod['klay_getChainConfig']
```

Comment:
Test fail is not because of the release on cypress/baobab. You need to delete the **chainConfig** in the deprecated file.

Reply:
It was because of missing klay_getChainConfig in types/packages/caver-rtm/src/index.d.ts. 
Thank you.